jobserver/views/job_requests.py

```diff
@@ -103,9 +104,18 @@ def dispatch(self, request, *args, **kwargs):
             messages.error(request, msg)
             return redirect(self.workspace)
 
+        # some backends might need to be disabled.  This view only uses
+        # backends the user can see so we look them up here, removing the
+        # relevant ones from the QS before we check if there are any below.
+        # The form, in get_form_kwargs, will also use the backends constructed
+        # here to be consistent.
+        self.backends = request.user.backends.all()
+        if settings.DISABLE_CREATING_JOBS:
+            self.backends = self.backends.exclude(Q(slug="emis") | Q(slug="tpp"))
```

Comment:
I understand why it's been done this way, but I dislike that we need to know that DISABLE_CREATING_JOBS affects only TPP and EMIS. I don't have a better solution though and hopefully it will only be here for a short time!